src/models/usersModel.js

```diff
@@ -0,0 +1,97 @@
+const db = require('./db');
+
+const createUser = async (params) => {
```

Comment:
We can add some debug message in userModels too. something like `log.debug("Running createUsesql = " + mysql.format(sql, vals));`. Our project is pretty small but these debug messages save a lot of time in the future.